graphql/init.lua

```diff
@@ -1 +1 @@
-return {VERSION = 'unknown'}
+return {_VERSION = require('graphql.version')}
```

Comment:
CI code for automatic version generation still exist. You need to remove it first.

https://github.com/tarantool/graphql/commit/0caab752868826f4367dbcf0cf71ecf6089a8f60